### ISSUE TITLE
dockerTools.mergeImages has a confusing name

### ISSUE THREAD

**Issue by divanorama** [NixOS/nixpkgs#137301]
### Describe the bug
One may think that `mergeImages` can combine two docker images into one, which makes some sense since store paths don't conflict. But it doesn't do that and rather creates an archive of several images bundled together that can all be loaded with single `docker load` command.

Maybe a name like `packageMultipleImages` or `bundleMultipleImages` would make the purpose more clear.

### Notify maintainers
@roberth

**Comment by roberth** [comment]
I agree. I like `dockerTools.bundleImages`. `Multiple` seems a bit redundant.

Iirc, it hasn't been around for very long, so the alias and deprecation won't even affect that many.
